README.md

```diff
@@ -347,8 +349,8 @@ stable versions - as soon as all Airflow dependencies support building, and we s
 building and testing the OS version. Approximately 6 months before the end-of-life of a previous stable
 version of the OS, Airflow switches the images released to use the latest supported version of the OS.
 For example since ``Debian Buster`` end-of-life was August 2022, Airflow switched the images in `main` branch
-to use ``Debian Bullseye`` in February/March 2022. The version was used in the next MINOR release after
-the switch happened. In case of the Bullseye switch - 2.3.0 version used ``Debian Bullseye``.
+to use ``Debian Bookworm`` in February/March 2022. The version was used in the next MINOR release after
+the switch happened. In case of the Bookworm switch - 2.3.0 version used ``Debian Bookworm``.
```

Comment:
I think we need to change date and version.
As well as previous line pointed to Buster (Debian 10)

Reply:
Fixed.